pkgs/by-name/dc/dclock/package.nix

```diff
@@ -35,11 +35,11 @@ haskellPackages.mkDerivation {
     process
   ];
 
-  meta = {
+  meta = with lib; {
     description = "Decimal clock that breaks your day into a 1000 decimal minutes";
     homepage = "https://github.com/travgm/dclock";
     license = lib.licenses.mit;
-    maintainers = with lib.maintainers; [ travgm ];
+    maintainers = with maintainers; [ travgm ];
```

Comment:
```suggestion
    maintainers = with lib.maintainers; [ travgm ];
```

Reply:
Not sure why it is still failing, every example and package I have checked has it this way as of my last update.

Reply:
according to this for haskell packages how it is now is correct https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/haskell.section.md#chap-meta